src/GuzzleClientFactory.php

```diff
@@ -80,20 +73,20 @@ public function getClient($baseUrl, array $options = [])
         return $this->initClient($baseUrl, $options);
     }
 
-    private function createDefaultDecider($maxRetries)
+    private function createDefaultDecider(int $maxRetries): callable
```

Comment:
Prekvapuje me, ze te PHPstan nenuti tu `callbable` natypovat.

Reply:
🤷 však callable je typ ne?

Reply:
No myslel jsem jako `@return callable(int, RequestInterface, ?ResponseInterface = null, ?Throwable = null): bool`. A ted jsem si vsiml druhe prekvapujici veci, ze mu nevadi, ze u `$retries` a `$error` nemas typ :thinking: Cim dal zajimavejsi.

Reply:
hmm, to retries a error je zajimavy

Reply:
ten callable chápu, v Guzzle je to taky callable